### PR TITLE
get edge TEdgeCustom from node edges implementation

### PR DESCRIPTION
**Pull request by ridicoulous**
#36 may be, I think, closed by this PR.

### CODE REVIEW

File: src/Dijkstra.NET/Graph/Node.cs [hunk 1]
```diff
@@ -51,7 +51,17 @@ public void EachEdge(Edge edge)
...
         }
-        
+        /// <summary>
+        /// Get custom info from node edges by edge key
+        /// </summary>
+        /// <param name="nodeEdgeKey">edge key</param>
+        /// <returns>TEdgeCustom</returns>
+        public TEdgeCustom GetEdgeCustom(int nodeEdgeKey)
+        {
+            return _edges.FirstOrDefault(c => c.Node.Key == nodeEdgeKey).Item;
```
[the remark this answers is not in the document]
Comment (ridicoulous) [reply src/Dijkstra.NET/Graph/Node.cs]: Thank you for comment. I can fix it, but it is not necessary if _edges will be public at next release, right?

[the remark this answers is not in the document]
Comment (matiii) [reply src/Dijkstra.NET/Graph/Node.cs]: INode interface should has this method as well.

[the remark this answers is not in the document]
Comment (matiii) [reply src/Dijkstra.NET/Graph/Node.cs]: Yep, edges will be public.

[the remark this answers is not in the document]
Comment (ridicoulous) [reply src/Dijkstra.NET/Graph/Node.cs]: - [ ] INode interface

- [ ] public Edges

**Pull request merged**